src/Tests/Helpers/Extensions.cs

```diff
@@ -0,0 +1,69 @@
+namespace SFA.DAS.Funding.SystemAcceptanceTests.Helpers;
+
+public static class Extensions
+{
+    public static void ShouldHaveCorrectFundingPeriods(this List<DeliveryPeriod> periods, int numberOfInstalments, short firstDeliveryPeriod, short firstDeliveryAcademicYear)
+    {
+        if (firstDeliveryPeriod > 12) throw new ArgumentException("Period can't be greater than 12 can it?", nameof(firstDeliveryPeriod));
+
+        var expectedPeriod = firstDeliveryPeriod;
+        var expectedAcademicYear = firstDeliveryAcademicYear;
+
+        List<(short Period, short AcademicYear)> expected = new();
+
+        for (var i = 0; i < numberOfInstalments; i++)
+        {
+            expected.Add(new ValueTuple<short, short>(expectedPeriod, expectedAcademicYear));
+            if (expectedPeriod == 12)
+            {
+                expectedPeriod = 1;
+                expectedAcademicYear = GetNextAcademicYear(expectedAcademicYear);
+            }
+            else expectedPeriod++;
+        }
+
+        var actual  = periods
+            .Select(c => new { c.Period, c.AcademicYear})
+            .AsEnumerable()
+            .Select(c => new Tuple<short, short>(c.Period, c.AcademicYear))
+            .ToList();
+
+        actual.Should().BeEquivalentTo(expected);
+    }
+
+    private static short GetNextAcademicYear(short expectedAcademicYear)
+    {
+        var firstPart = expectedAcademicYear / 100 + 1;
+        var secondPart = expectedAcademicYear % 100 + 1;
+        return (short)(firstPart * 100 + secondPart);
+    }
```

Comment:
I ended up writing a little console app to test this method, I am on the fence as to whether it's more readable like this or using substring method

Reply:
if you're on the fence then best choose the fastest method, which should be this one ;)